torch/quantization/fake_quantize.py

```diff
@@ -59,22 +59,30 @@ def __init__(self, observer=MovingAverageMinMaxObserver, quant_min=0, quant_max=
         self.register_buffer('zero_point', torch.tensor([0]))
         self.dtype = self.activation_post_process.dtype
         self.qscheme = self.activation_post_process.qscheme
-        self.ch_axis = self.activation_post_process.ch_axis if hasattr(self.activation_post_process, 'ch_axis') else None
+        self.ch_axis = self.activation_post_process.ch_axis \
```

Comment:
this is because `fake_quantize_per_channel_affine` expects an integer axis.  Axis is enforced at runtime to be >0 in that function.